common/utils/ga.ts

```diff
@@ -1,14 +1,8 @@
-import ReactGA from 'react-ga';
-
-export type GaEvent = {
-  category: string;
-  action: string;
-  label?: string;
-  value?: number;
-  nonInteraction?: boolean;
-  transport?: 'beacon';
-};
+import ReactGA, { EventArgs as GaEvent } from 'react-ga';
 
+// TODO this wrapper function is redundant but pervasive; consider getting rid of it
```

Comment:
get rid of as in replace with an inline call to `ReactGA.event(gaEvent)`?

Reply:
I started doing this but its usage is _extensive_ and since we will be getting rid of UA I thought it probably unnecessary

Reply:
Oh sorry I misread this comment - yes, correct